src/Agent.Worker/TestResults/ResultsCommandExtension.cs

```diff
@@ -391,6 +391,7 @@ private void TriggerCoverageMergeJob(List<string> resultFilesInput, IExecutionCo
                         }
                     }
                 }
+                connection.Dispose();
```

Comment:
Why not `using`? In case of exception `connection` will not be disposed

Reply:
My bad for missing it. I have updated to using statement.